.github/workflows/_pr_entrypoint.yaml

```diff
@@ -140,6 +140,7 @@ jobs:
         env:
           PROFILE: ${{ matrix.profile }}
           ENABLE_COVER_COMPILE: 1
+          NEW_MIX_BUILD: 1
         run: |
           make ensure-rebar3
           make ${PROFILE}-compile test-compile
```

Comment:
probably should split this into 2 concurrent jobs with different artifacts (one for tests, one for prod builds)